lib/stripe/credit_card_clone_destroyer.rb

```diff
@@ -17,7 +17,9 @@ def destroy_clones(card)
         next unless customer_id
 
         customer = Stripe::Customer.retrieve(customer_id, stripe_account: stripe_account)
-        customer&.delete unless customer.deleted?
+        next unless customer
+
+        customer.delete unless customer.deleted?
```

Comment:
wouldnt this do the trick?
        `customer.delete if customer && !customer.deleted?`